src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: fit-content;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-size: cover;
+}
+
+.stars--0 .stars__star:nth-child(-n+0),
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
```

Comment:
The `:nth-child` selectors are incorrectly configured. For example, `.stars--0 .stars__star:nth-child(-n+0)` will not select any elements because `-n+0` results in no valid index. Similarly, `.stars--1 .stars__star:nth-child(-n+1)` will only select the first child. Consider adjusting these selectors to correctly reflect the intended number of active stars. For instance, `.stars--1 .stars__star:nth-child(-n+1)` should be `.stars--1 .stars__star:nth-child(-n+2)` to select the first two stars.